src/main/java/org/dependencytrack/resources/v1/ConfigPropertyResource.java

```diff
@@ -137,4 +135,34 @@ public Response updateConfigProperty(List<ConfigProperty> list) {
         }
         return Response.ok(returnList).build();
     }
+
+    @POST
+    @Path("upsert")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(
+            value = "Creates/updates a config property",
+            response = ConfigProperty.class
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 401, message = "Unauthorized")
+    })
+    @PermissionRequired(Permissions.Constants.SYSTEM_CONFIGURATION)
+    public Response upsertsConfigProperty(ConfigProperty json) {
```

Comment:
Just checked the frontend, this endpoint is not used by it and can be removed. I think this may be a leftover from a previous PR?